keps/sig-cli/20190308-kustomize-apply.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Add apply command to kustomize
+authors:
+  - "@monopole"
+owning-sig: sig-cli
+participating-sigs:
+  - sig-cli
+  - sig-api
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-03-08
+last-updated: 2019-03-08
+status: provisional
+see-also:
+  - "/keps/sig-api-machinery/0006-apply.md"
+---
+
+
+[DAM]: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/declarative-application-management.md
+[SSA-KEP]: /keps/sig-api-machinery/0006-apply.md
+[SSA-PR]: https://github.com/kubernetes/kubernetes/pull/72946
```

Comment:
probably should use 72947 instead?  (off-by-one error)

Reply:
fixed, thanks